docs/shared/guides/module-federation/dynamic-mfe-angular.md

```diff
@@ -461,6 +456,10 @@ nx serve dashboard --devRemotes=login
 
 Should result in the same behaviour as before, except that our Dashboard application is waiting until runtime to find out the deployed location of our Login application.
 
+{% callout type="note" title="Check out the example repo!" %}
```

Comment:
Can you use the dedicated component to embed example repositories please?
You can find how to do it in [the readme over here](https://github.com/nrwl/nx/blob/master/docs/README.md#github-repositories)